README.md

```diff
@@ -81,52 +90,47 @@ We have made AWS credentials available in the following file, with the appropria
 ### 1. Data Extraction (if applying for a Data Engineering Position)
 Use the [AWS S3 SELECT](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-glacier-select-sql-reference-select.html) command to read in the H3 resolution 8 data from `city-hex-polygons-8-10.geojson`. Use the `city-hex-polygons-8.geojson` file to validate your work.
 
-Please log the time taken to perform the operations described, and within reason, try to optimise latency and computational resources used.
+Please log the time taken to perform the operations described, and within reason, try to optimise latency and computational resources used. Please also note the comments above about the nature of the code that we expect.
 
 ### 2. Initial Data Transformation (if applying for a Data Engineering and/or Science Position)
-Join the file `city-hex-polygons-8.geojson` to the service request dataset, such that each service request is assigned to a single H3 hexagon. Use the `sr_hex.csv` file to validate your work.
+Join the equivalent of the contents of the file `city-hex-polygons-8.geojson` to the service request dataset, such that each service request is assigned to a single H3 resolution level 8 hexagon. Use the `sr_hex.csv` file to validate your work.
 
 For any requests where the `Latitude` and `Longitude` fields are empty, set the index value to `0`.
 
-Include logging that lets the executor know how many of the records failed to join, and include a join error threshold above which the script will error out. Please also log the time taken to perform the operations described, and within reason, try to optimise latency and computational resources used.
+Include logging that lets the executor know how many of the records failed to join, and include a join error threshold above which the script will error out. Please motivate why you have selected the error threshold that you have. Please also log the time taken to perform the operations described, and within reason, try to optimise latency and computational resources used.
 
 ### 3. Descriptive Analytic Tasks (if applying for a Data Analyst Position)
 Please use the `sr_hex_truncated.csv` dataset to address the following.
 
 Please provide the following:
-1. Provide a visual answer to the question "which areas and request types should Electricity concentrate on to reduce the overall volume of their requests".
-2. Provide a working prototype dashboard for monitoring progress in reducing Electricity service request volume per area, and per type.
+1. An answer to the question "In which suburbs should Water and Sanitation concentrate their infrastructure improvement efforts?". Please motivate how you related the data provided to infrastructure issues.
```

Comment:
JB notes:  I suggest making the question more open ended.  My suggestion below, in my opinion, is a more advanced test for the candidate's ability to explore and derive value from a dataset that they are not familiar with.  

1. Identify value adding insights from the dataset provided with regards to electricity provision within the city.  
2. Develop a dashboard with visuals that draws the eye to performance pain points.  Add a note to each visual as to what business value it is adding in the context of the dashboard. (Extra credit for visuals that adapt automatically when a problem is addressed.  You may assume that new data will come in on a daily basis)